code/obj/item/misc_weapons.dm

```diff
@@ -38,7 +38,7 @@
 	flags = FPRINT | TABLEPASS | NOSHIELD | USEDELAY
 	tool_flags = TOOL_CUTTING
 	is_syndicate = 1
-	mats = 18
+	mats = 35
```

Comment:
Changing this to also some kind of specific/custom mat (even just bohrum or something) would be ideal, I think

Reply:
ye